app_base.htm

```diff
@@ -0,0 +1,51 @@
+<%inherit file="/homepage/templates/base.htm" />
+
+
+<%block name="navbar_links">
+<!-- <li class=" ${'active' if request.dmp.page =='productList' else ''}"><a href="/manager/productList">Products</a></li>
+<li class=" ${'active' if request.dmp.page =='createProduct' else ''}"><a href="/manager/createProduct">Create Product</a></li> -->
+</%block>
+
+
+<%block name="content_left">
+  <%! from catalog import models as cmod %>
+  <% categories = cmod.Category.objects.order_by('name') %>
+  <div class="content">
+    <ul id="category">
+      <li class="${'active' if (category is None) else ''}"><a href="/catalog/index/">All Products</a></li>
+      %for c in categories:
+      <li class="${'active' if (category is not None and c.id == category.id) else ''}">
```

Comment:
indent here to make the code more readable for the python for loop

Reply:
that is a great idea, it will make it much more readable